Description: Hard-code apreq2-config --prefix
 apreq2-config will leak information about the build environment.
 Hard-code what --prefix returns, so it's at least slightly more sane.
Author: Steinar H. Gunderson <sesse@debian.org>
Origin: vendor
Bug-Debian: http://bugs.debian.org/323108
Forwarded: not-needed
Last-Update: 2010-02-28

--- libapreq2-2.12.orig/apreq2-config.in
+++ libapreq2-2.12/apreq2-config.in
@@ -22,7 +22,7 @@
 
 PATH=/bin:/usr/bin:/usr/local/bin
 
-prefix="@prefix@"
+prefix="/usr"
 exec_prefix="@exec_prefix@"
 bindir="@bindir@"
 libdir="@libdir@"
